src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java

```diff
@@ -1,39 +1,201 @@
+                    judicialOfficeAuthorisation.setCreatedDate(handleNull((Timestamp)authorisationMap
+                            .get("created_date")));
+                    judicialOfficeAuthorisation.setLastUpdated(handleNull((Timestamp)authorisationMap
+                            .get("last_updated")));
+                    judicialOfficeAuthorisation.setLowerLevel((String)authorisationMap.get("lower_level"));
+                    return judicialOfficeAuthorisation;
+                }).collect(Collectors.toList());
+
+        //size check
+        List<JudicialOfficeAuthorisation> expectedAuthorisations = getFileAuthorisationObjectsFromCsv(file[2]);
+        assertEquals(judicialAuthorisationList.size(), expectedAuthorisations.size());
+        //exact field checks
+        Assertions.assertThat(actualAuthorisations).usingFieldByFieldElementComparator()
+                .containsAll(expectedAuthorisations);
+    }
+
+    static List<JudicialOfficeAuthorisation> getFileAuthorisationObjectsFromCsv(String inputFilePath)  {
+        List<JudicialOfficeAuthorisation> authorisations = new LinkedList<>();
+        try {
+            File file = ResourceUtils.getFile(inputFilePath);
+            InputStream inputStream = new FileInputStream(file);
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+            authorisations = bufferedReader.lines().skip(1).map(line -> mapJudicialOfficeAuthorisation(line))
+                    .collect(Collectors.toList());
+            bufferedReader.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return authorisations;
+    }
+
+    static JudicialOfficeAuthorisation mapJudicialOfficeAuthorisation(String line) {
+        List<String> columns = Arrays.asList(line.split("\\,", -1));
+        JudicialOfficeAuthorisation judicialOfficeAuthorisation = new JudicialOfficeAuthorisation();
+        judicialOfficeAuthorisation.setElinksId(handleNull(columns.get(0), false));
+        judicialOfficeAuthorisation.setJurisdiction(handleNull(columns.get(1), false));
+        judicialOfficeAuthorisation.setTicketId(isBlank(columns.get(2)) ? null : Long.parseLong(columns.get(2)));
+        judicialOfficeAuthorisation.setStartDate(handleNull(columns.get(3), true));
+        judicialOfficeAuthorisation.setEndDate(handleNull(columns.get(4), true));
+        judicialOfficeAuthorisation.setCreatedDate(handleNull(columns.get(5), true));
+        judicialOfficeAuthorisation.setLastUpdated(handleNull(columns.get(6), true));
+        judicialOfficeAuthorisation.setLowerLevel(handleNull(columns.get(7), false));
+        return judicialOfficeAuthorisation;
+    }
+
+    static String handleNull(String fieldValue, boolean timeStampField) {
+        if (isBlank(fieldValue) && !timeStampField) {
+            return StringUtils.EMPTY;
+        } else if (timeStampField && isBlank(fieldValue)) {
```

Comment:
https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/ObjectUtils.html#defaultIfNull(T,%20T)

you can use this.

Reply:
Yes checked the methods, but somehow doesn't fit for the custom condition.
Here for String field i want EMPTY value  and for timestamp field wanted null value, this cant be achieved via apache util method.